lib/elixir/lib/system.ex

```diff
@@ -369,9 +369,39 @@ defmodule System do
   """
   @spec get_env(binary) :: binary | nil
```

Comment:
Given it uses `List.to_string`, yes. It may be the wrong value if the list is a list of bytes instead of a list of codepoints but it will still be valid UTF-8.